### PR TITLE
Add brew installation option to documentation.

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

### CODE REVIEW

File: content/spin/install.md
Comment (mikkelhegn) [inline content/spin/install.md:25]: I'd suggest removing the 'now' as this speaks to this very moment :-)

File: content/spin/install.md
Comment (mikkelhegn) [inline content/spin/install.md:61]: I'd suggest removing the 'now' as this speaks to this very moment :-)

**Review by mikkelhegn — APPROVED** [review]
Small nit, good to go if you don't want to make the change :-)

File: content/spin/install.md
Comment (kate-goldenring) [inline content/spin/install.md:27]: I'd recommend breaking this into two steps to emphasize that they first need to install the Fermyon tap, a repository of all the Fermyon Homebrew packages, and then isntall spin with `fermyon/tap/spin`. See Hashicorp for example: https://developer.hashicorp.com/nomad/docs/install.

We also should add an instruction on how to upgrade to latest:
`brew upgrade fermyon/tap/nomad`

File: content/spin/install.md
Comment (kate-goldenring) [inline content/spin/install.md:25]: We should say what that directory is (where plugins and templates are installed), namely `$HOMEBREW_PREFIX/fermyon-spin`

Comment (kate-goldenring) [inline content/spin/install.md:25]: There is a [homebrew core formula for a verification tool called spin](https://formulae.brew.sh/formula/spin). We should call out that a `brew install spin` will not install our Spin. They need to install the spin from our tap. (another reason i recommend breaking out the steps into a tap install and then a project install to emphasize the tap).

**Review by kate-goldenring — CHANGES_REQUESTED** [review]
Before we merge this i'd really like to emphasize the tap install in order to prevent unwanted `brew install spin`s

File: content/spin/install.md
Comment (itowlson) [reply content/spin/install.md:25]: Agree though this would be best done i https://developer.fermyon.com/spin/cache rather than here

File: content/spin/install.md
Comment (itowlson) [reply content/spin/install.md:27]: Again, I agree about documenting the upgrade command, but with the current docs structure this would go better on https://developer.fermyon.com/spin/upgrade

Comment (tpmccallum) [inline content/spin/install.md:27]:
```suggestion
Add Spin to the list of formulae that Homebrew tracks, updates, and installs from:

<pre class="bash spin-install" id="spin-install-linux"><code>$ brew tap fermyon/tap</code></pre>

Install Spin:

<pre class="bash spin-install" id="spin-install-linux"><code>$ brew install fermyon/tap/spin</code></pre>

> Note: `brew install spin` will **not** install Fermyon's Spin framework. Fermyon Spin is accessed from the `fermyon` tap, as shown above.

File: content/spin/install.md
Comment (tpmccallum) [inline content/spin/install.md:63]:
```suggestion
Add Spin to the list of formulae that Homebrew tracks, updates, and installs from:

<pre class="bash spin-install" id="spin-install-macos"><code>$ brew tap fermyon/tap</code></pre>

Install Spin:

<pre class="bash spin-install" id="spin-install-macos"><code>$ brew install fermyon/tap/spin</code></pre>

> Note: `brew install spin` will **not** install Fermyon's Spin framework. Fermyon Spin is accessed from the `fermyon` tap, as shown above.

File: content/spin/install.md
Comment (tpmccallum) [reply content/spin/install.md:27]: Please see new PR that updates the upgrade page
https://github.com/fermyon/developer/pull/827

File: content/spin/install.md
Comment (tpmccallum) [reply content/spin/install.md:25]: Thanks, done.

File: content/spin/install.md
Comment (tpmccallum) [reply content/spin/install.md:61]: Thanks, done.

File: content/spin/install.md
Comment (tpmccallum) [reply content/spin/install.md:25]: Thanks, note added.

**Comment by tpmccallum** [comment]
Thanks Kate, we have emphasize the Fermyon `tap` as part of the instructions in order to prevent a user from accidentally installing the wrong Spin from Brew.

File: content/spin/install.md
Comment (kate-goldenring) [inline content/spin/install.md:27]: technically that is not what this command does. Rather it fetches Fermyon's homebrew tap
```suggestion
Install the Fermyon tap, which Homebrew tracks, updates, and installs Spin from:
```

**Comment by karthik2804** [comment]
A non-blocking question out of curiosity - why are we writing raw HTML instead of markdown here?

File: content/spin/install.md
Comment (kate-goldenring) [inline content/spin/install.md:71]:
```suggestion
Install the Fermyon tap, which Homebrew tracks, updates, and installs Spin from:
```

**Review by kate-goldenring — COMMENTED** [review]
Thanks for updating. One small correction

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**
